app/components/comment-item.js

```diff
@@ -1,5 +1,49 @@
 import Ember from 'ember';
 
 export default Ember.Component.extend({
-  classNames: ['comment-item']
+  classNames: ['comment-item'],
+  classNameBindings: ['isEditing:editing'],
+
+  session: Ember.inject.service(),
+
+  currentUserId: Ember.computed.alias('session.session.authenticated.user_id'),
```

Comment:
We may want to refactor this later with the `currentUser` object I'm introducing in #76.